src/extension/module.js

```diff
@@ -1546,6 +1549,108 @@
     }
   }
 
+  function getTimeAgo(dateParam) {
+    if (!dateParam) {
+      return ' --- ';
+    }
+    const date = typeof dateParam === 'object' ? dateParam : new Date(dateParam);
+
+    const today = new Date();
+    const yesterday = new Date(today - 86400000); // 86400000 = ms in a day
+    const seconds = Math.round((today - date) / 1000);
+    const minutes = Math.round(seconds / 60);
+    const isToday = today.toDateString() === date.toDateString();
+    const isYesterday = yesterday.toDateString() === date.toDateString();
+    const isThisYear = today.getFullYear() === date.getFullYear();
+
+    if (seconds < 5) {
+      return '<span class="now">';
+    } else if (seconds < 120) {
+      return `${seconds} <span class="seconds-ago">`;
+    } else if (minutes < 60) {
+      return `${minutes} <span class="minutes-ago">`;
+    } else if (isToday) {
+      return `<span class="today"> ${date.toLocaleTimeString([], { timeStyle: 'medium' })}`;
+    } else if (isYesterday) {
+      return `<span class="yesterday"> ${date.toLocaleTimeString([], { timeStyle: 'medium' })}`;
+    } else if (isThisYear) {
+      return date.toLocaleString([], {
+        month: 'short',
+        day: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+      });
+    }
+
+    return date.toLocaleString([], {
+      year: 'numeric',
+      month: 'short',
+      day: 'numeric',
+      hour: 'numeric',
+      minute: 'numeric',
+    });
+  }
+
+  function updateModifiedDates(sk) {
+    const infoPlugin = sk.get('info');
+    if (!infoPlugin) {
+      return;
+    }
+
+    const editEl = infoPlugin.querySelector('.edit-date');
+    const previewEl = infoPlugin.querySelector('.preview-date');
+    const publishEl = infoPlugin.querySelector('.publish-date');
+
+    const { status } = sk;
+    const editLastMod = (status.edit && status.edit.lastModified) || null;
+    const previewLastMod = (status.preview && status.preview.lastModified) || null;
+    const liveLastMod = (status.live && status.live.lastModified) || null;
+
+    editEl.innerHTML = getTimeAgo(editLastMod);
+    previewEl.innerHTML = getTimeAgo(previewLastMod);
+    publishEl.innerHTML = getTimeAgo(liveLastMod);
```

Comment:
Add a `null` check and display an empty string instead.

Reply:
`getTimeAgo` only returns strings